vars/rollingUpgradePipeline.groovy

```diff
@@ -84,6 +84,10 @@ def call(Map pipelineParams) {
 
                                                         export SCT_CLUSTER_BACKEND=${params.backend}
 
+                                                        if [[ -n "${params.aws_region ? params.aws_region : ''}" ]] ; then
+                                                            export SCT_REGION_NAME=${aws_region}
```

Comment:
Shouldn't it be "params.aws_region" ? I don't see definition of the "aws_region" variable.

Reply:
i think not, as if we don't have `params.aws_region` or if it is an empty string, we take the value from the pipeline field called `aws_region` here:

```
string(defaultValue: "${pipelineParams.get('aws_region', 'eu-west-1')}",
               description: 'Supported: us-east-1|eu-west-1|eu-west-2|eu-north-1|random (randomly select region)',
               name: 'aws_region')
```